core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -180,9 +178,8 @@ class Partition(val topicPartition: TopicPartition,
   @volatile private var leaderEpochStartOffsetOpt: Option[Long] = None
   @volatile var leaderReplicaIdOpt: Option[Int] = None
   @volatile var inSyncReplicas: Set[Int] = Set.empty[Int]
-  // Includes all valid broker ids (@see Request::isValidBrokerId) that contain logs. Doesn't contain
-  // future log replica id
-  @volatile var allReplicaIds: scala.collection.mutable.Set[Int] = scala.collection.mutable.Set(localBrokerId)
+  // Includes all valid broker ids (@see Request::isValidBrokerId) that were assigned to this topic partition.
+  @volatile var allReplicaIds = Seq.empty[Int]
```

Comment:
nit: we are a little inconsistent with naming. Should we change `inSyncReplicas` to `inSyncReplicaIds`?

Reply:
Done.